src/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,73 @@
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
+import { faCaretDown, faTools } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { pages } from '../../assets/pages';
+
+import './NavBar.css';
+import NavBarTools from './NavBarTools';
+import NavBarLogo from './NavBarLogo';
+
+interface PlatformType {
```

Comment:
why  PlatformType defined here and not in the enum file?

Reply:
One-time use and no enum file (currently)
Did something else